components/TextFields/src/MDCMultilineTextField.m

```diff
@@ -481,7 +481,7 @@ - (void)updateTrailingViewAlpha {
 - (CGFloat)trailingViewAlpha {
   // The trailing view has the same behavior as .rightView in UITextField: It has visual precedence
   // over the clear button.
-  CGFloat trailingViewAlpha = self.trailingView.alpha;
+  CGFloat trailingViewAlpha;
```

Comment:
This isn't correct... if `self.trailingViewMode` is garbage (enum has an illegal value), then you'll return an uninitialized value, which could be anything (NaN, -22, etc.).

If you want to keep this change, add a default clause to the switch statement and cover debug/release contingencies, like this:

~~~
switch (self.trailingViewMode) {
  ...
  default:
    assert(NO, @"Invalid enumeration value %i.", self.trailingViewMode);
    trailingViewAlpha = self.trailingViewAlpha;
}

Reply:
+1 ajsecord's remark, but please use NSAssert() instead of assert().

Reply:
Done.

Although, it's not necessarily incorrect atm since current switch statement is exhaustive. `UITextFieldViewMode` has only 4 possible values. However, it does not protect from future additions to this enum or a programmer error initializing to an invalid value.